### DISCUSSION TITLE
Does the input shape affect performance?

### DISCUSSION THREAD

**Discussion by LinGeLin** [openvinotoolkit/openvino#27055] [Q&A]
Does the input shape affect performance? For example, if there are two similar models, one of which has dynamic inputs that can set the batch_size, so the first dimension of the shape is the same, while the other model has some inputs with fixed shapes, so the first dimension of the input shape is not the same. Will there be differences in performance between the two scenarios?

**Comment by azhai219** [comment]
@LinGeLin Yes, it does. In the most cases, static shape model is faster than dynamic shape model.
